.github/workflows/rust.yml

```diff
@@ -47,12 +47,32 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: build
-          args: --all --all-targets
+          args: --workspace --all-targets
       - uses: actions-rs/cargo@v1
         with:
           command: test
           args: --all-features --no-fail-fast
 
+  test-integration-ignored:
+    runs-on: ubuntu-latest
+    services:
+      tendermint:
+        image: interchainio/tendermint
+        ports:
+          - 26656:26656
+          - 26657:26657
+          - 26660:26660
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: -p tendermint --test integration --no-fail-fast -- --ignored
```

Comment:
Note to myself: this won't run the light client tests yet.

Reply:
Yes, I actually didn't find those tests. Also, these tests are not mandatory to merge something in GitHub.